plugins.json

```diff
@@ -3608,5 +3608,17 @@
         "requires": ">=24.10.0"
       }
     ]
+  },
+  {
+    "id": "nf-dataflow",
+    "releases": [
+      {
+        "version": "0.0.1",
+        "date": "2025-04-02T12:31:22.502790223+02:00",
+        "url": "https://github.com/nextflow-io/nf-dataflow/releases/download/0.0.1/nf-dataflow-0.0.1.zip",
+        "requires": ">=24.0.0",
```

Comment:
This version looks wrong, if it works with any 24.x version then I would set min version to 24.04.0